### ISSUE TITLE
Fenwick Tree

### ISSUE THREAD

**Issue by Kanhakhatri065** [ephremdeme/data-structure-and-algorithms#239]
Implement Fenwick tree in any language.

**Comment by anurag-bhatt** [comment]
I can do it in C++ if you want

**Comment by kavya1229** [comment]
I will do it in python, please assign this to me.

**Comment by Kanhakhatri065** [comment]
@anurag-bhatt do it in c++.

**Comment by Kanhakhatri065** [comment]
@kavya1229 wait let me view your PR.

**Comment by khannakshat7** [comment]
@Kanhakhatri065 Can I do it in Python? Pl. assign it to me

**Comment by Kanhakhatri065** [comment]
@khannakshat7 No, I have assigned C++ and python implementation to other contributors but you are free to implement in any other language.
